mk_robotics/ws_manager/WebSocketManager.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <drogon/WebSocketConnection.h>
+
+#include <vector>
+#include <algorithm>
+#include <string>
+
+namespace mk_robotics
+{
+    namespace ws_manager
+    {
+        class WebSocketManager
+        {
+        public:
+            template <typename F>
+            void forEach(F &&f)
+            {
+                std::for_each(_clients.begin(), _clients.end(), f);
+            }
+            
+            void notifyAll(const std::string &msg);
+            void add(const drogon::WebSocketConnectionPtr &);
+            
+
+        private:
+            std::vector<drogon::WebSocketConnectionPtr> _clients;
```

Comment:
Are we trying to unify code conventions in motor_driver and drogon repository?  I started using PascalCase in methods and underscore after class variable name

Reply:
Good point